lib/core/PlaylistManager.ts

```diff
@@ -93,16 +73,19 @@ class PlaylistManager {
 
     const payload = {
       playlistId: playlist_id,
-      actions: []
+      actions: [] as {
+                action: string;
+                setVideoId: string;
+            }[]
```

Comment:
Perhaps extract the type to make this a tiny bit more clear?

Reply:
These types aren't used anywhere else, only in that one function, so probably unnecessary to extract.

Reply:
Alright! Seems good